src/labeler.ts

```diff
@@ -40,6 +40,23 @@ export async function run() {
 
     const labels: string[] = [];
     const labelsToRemove: string[] = [];
+
+    if (changedFiles.length === 0) {
+      core.debug(`exiting early because pr #${prNumber} has no changed files.`);
+
+      if (syncLabels) {
+        for (let label of pullRequest.labels) {
+          if (label.name !== undefined) {
```

Comment:
I think we can change it to the same check if (label.name) {

Reply:
Confirmed that name can't be blank at creation time. (returns 422)

`{"name":["Name can't be blank"],"message":"Name can't be blank"}`